exo/inference/inference_engine.py

```diff
@@ -15,6 +15,10 @@ async def infer_prompt(self, request_id: str, shard: Shard, prompt: str, image_s
   async def infer_tensor(self, request_id: str, shard: Shard, input_data: np.ndarray, inference_state: Optional[str] = None) -> Tuple[np.ndarray, str, bool]:
     pass
 
+  @abstractmethod
+  def benchmark_tflops(self):
```

Comment:
This is going to need to be async otherwise it will block the main thread

Reply:
Fixed!